src/backtick/engine.clj

```diff
@@ -116,7 +130,7 @@
             (db/queue-insert! {:name (:name payload)
                                :state "running"
                                :priority (to-sql-time (time/now))
-                               :data [[]]})
+                               :data (prn-str [[]])})
```

Comment:
Yeah that's the hope... Is there any way we can exercise this code path in a test?